index.js

```diff
@@ -152,27 +135,17 @@ signInForm.addEventListener("input", () => {
 })
 
 //enabling or disabling buttons
-function enableBtns() {
-    buttonsArr.forEach(button => {
-        button.disabled = false;
-    });
-}
-
-export function disableBtns() {
-    buttonsArr.forEach(button => {
-        button.disabled = true;
-    });
+export function toggleDisableBtns(...buttonArrays){
+    buttonArrays.forEach(buttonArray =>
```

Comment:
I thought we were moving the normal functions not arrow. Or is it arrow functions within normal functions?